transformers/algorithm.md

```diff
@@ -0,0 +1,40 @@
+# Algorithm
+This is the name given to the portion of a transformer that provides the analysis and/or transformation of data.
+
+Please read the [Transformers](https://github.com/AgPipeline/AgPipeline.github.io/blob/master/transformers/transformers.md) overview documentation for some additional context.
+
+In this document we will be providing an conceptual overview of the Algorithm code structure and its implementation framework for the UA makeflow solution.
+
+## Overview
+The use of the term Algorithm appears to be nebulous in that implementations can range from processing raw sensor data to performing time based analysis of derived data.
+The unifying principle for Algorithm implementation is the manipulation of data.
+For example, the Algorithm implementations for RGB processing in the AgPipeline includes converting the *.bin* files to a georeferenced *.tif* file, masking soil from the TIFFs, clipping images to plot boundaries, and calculating canopy cover from the plot-clipped TIFF data.
+
+By standardizing a minimal set of required file names and function signatures with return values, Algorithm implementations can more easily be used in different environments.
```

Comment:
are we standardizing file names? what are function signatures?

Reply:
At this point in time we're standardizing the file names. We can change that down the road, and Rob Kooper would like that as well since it'd mesh with the "Simple Extractor" code he has.